2021-10-18 19:18:01,874 DEBUG decos.py Была вызвана функция main c параметрами () , {}. Вызов из модуля __main__
2021-10-18 19:18:01,874 DEBUG decos.py Была вызвана функция config_load c параметрами () , {}. Вызов из модуля __main__
2021-10-18 19:18:01,875 DEBUG decos.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2021-10-18 19:18:01,875 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['H:/Программирование/Основной курс Python/Базы данных и PyQT/Project_Pyqt/QWW/server.py']
2021-10-18 19:18:01,877 DEBUG server.py Аргументы успешно загружены.
2021-10-18 19:18:02,024 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-10-18 19:18:48,064 INFO core.py Установлено соедение с ПК ('127.0.0.1', 63576)
2021-10-18 19:18:48,566 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63576)>,) , {}. Вызов из модуля common.utils
2021-10-18 19:18:48,566 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1634573928.0955143, 'user': {'account_name': 'vlad', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuQFBj9Hg1O01f47kTRlp\n88c45AoArd+JZ70EN92bpdbR/ZktTkETimyTNauaIo3iVYW/ptzdQLR1X9YGk3Jr\nfw+GkcCQUPpByqWECQ/LqudCi2k3IAQQu8Kh+1pdIhwhVJ/CRk1Pqt0UUYwTDBsN\ng7Lg0mfswhPHCQEKc41WAZjQsFWHkTCOPjEEOepQcC2gowzP0mlqJsl1a5e3zad7\nHCHs/debdWRInNvtYGnEi4Crx49Z2tkowjHEd/5Q6XKp+siDEXaqP5M9wAkbQSJp\nuvFXZGXOF1ipKa7iKiKCFguePWoXhIKotiCHX/2nYocyuxML1AI7MWmCH3NTrotu\nRQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-10-18 19:18:48,566 DEBUG core.py Start auth process for {'account_name': 'vlad', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuQFBj9Hg1O01f47kTRlp\n88c45AoArd+JZ70EN92bpdbR/ZktTkETimyTNauaIo3iVYW/ptzdQLR1X9YGk3Jr\nfw+GkcCQUPpByqWECQ/LqudCi2k3IAQQu8Kh+1pdIhwhVJ/CRk1Pqt0UUYwTDBsN\ng7Lg0mfswhPHCQEKc41WAZjQsFWHkTCOPjEEOepQcC2gowzP0mlqJsl1a5e3zad7\nHCHs/debdWRInNvtYGnEi4Crx49Z2tkowjHEd/5Q6XKp+siDEXaqP5M9wAkbQSJp\nuvFXZGXOF1ipKa7iKiKCFguePWoXhIKotiCHX/2nYocyuxML1AI7MWmCH3NTrotu\nRQIDAQAB\n-----END PUBLIC KEY-----'}
2021-10-18 19:18:48,569 DEBUG core.py Unknown username, sending {'response': 400, 'error': 'Пользователь не зарегистрирован.'}
2021-10-18 19:18:48,569 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63576)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}) , {}. Вызов из модуля common.utils
2021-10-18 19:19:20,887 INFO core.py Установлено соедение с ПК ('127.0.0.1', 63578)
2021-10-18 19:19:21,397 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>,) , {}. Вызов из модуля common.utils
2021-10-18 19:19:21,397 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1634573960.8989358, 'user': {'account_name': 'vlad', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuQFBj9Hg1O01f47kTRlp\n88c45AoArd+JZ70EN92bpdbR/ZktTkETimyTNauaIo3iVYW/ptzdQLR1X9YGk3Jr\nfw+GkcCQUPpByqWECQ/LqudCi2k3IAQQu8Kh+1pdIhwhVJ/CRk1Pqt0UUYwTDBsN\ng7Lg0mfswhPHCQEKc41WAZjQsFWHkTCOPjEEOepQcC2gowzP0mlqJsl1a5e3zad7\nHCHs/debdWRInNvtYGnEi4Crx49Z2tkowjHEd/5Q6XKp+siDEXaqP5M9wAkbQSJp\nuvFXZGXOF1ipKa7iKiKCFguePWoXhIKotiCHX/2nYocyuxML1AI7MWmCH3NTrotu\nRQIDAQAB\n-----END PUBLIC KEY-----'}}
2021-10-18 19:19:21,397 DEBUG core.py Start auth process for {'account_name': 'vlad', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuQFBj9Hg1O01f47kTRlp\n88c45AoArd+JZ70EN92bpdbR/ZktTkETimyTNauaIo3iVYW/ptzdQLR1X9YGk3Jr\nfw+GkcCQUPpByqWECQ/LqudCi2k3IAQQu8Kh+1pdIhwhVJ/CRk1Pqt0UUYwTDBsN\ng7Lg0mfswhPHCQEKc41WAZjQsFWHkTCOPjEEOepQcC2gowzP0mlqJsl1a5e3zad7\nHCHs/debdWRInNvtYGnEi4Crx49Z2tkowjHEd/5Q6XKp+siDEXaqP5M9wAkbQSJp\nuvFXZGXOF1ipKa7iKiKCFguePWoXhIKotiCHX/2nYocyuxML1AI7MWmCH3NTrotu\nRQIDAQAB\n-----END PUBLIC KEY-----'}
2021-10-18 19:19:21,398 DEBUG core.py Correct username, starting passwd check.
2021-10-18 19:19:21,420 DEBUG core.py Auth message = {'response': 511, 'bin': '24fda4a6eccc545dde9292a32a0c76ee565a15322e368c9d70ad94163b26b21109e6a30ab988069b213b37a56b1322e7653d41ff86ab42e6d3c515fa25553388'}
2021-10-18 19:19:21,420 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>, {'response': 511, 'bin': '24fda4a6eccc545dde9292a32a0c76ee565a15322e368c9d70ad94163b26b21109e6a30ab988069b213b37a56b1322e7653d41ff86ab42e6d3c515fa25553388'}) , {}. Вызов из модуля common.utils
2021-10-18 19:19:21,420 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>,) , {}. Вызов из модуля common.utils
2021-10-18 19:19:21,420 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>, {'response': 200}) , {}. Вызов из модуля common.utils
2021-10-18 19:19:22,058 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>,) , {}. Вызов из модуля common.utils
2021-10-18 19:19:22,058 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1634573961.4205694, 'account_name': 'vlad'}
2021-10-18 19:19:22,064 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>, {'response': 202, 'data_list': ['test1', 'test2', 'vlad']}) , {}. Вызов из модуля common.utils
2021-10-18 19:19:22,573 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>,) , {}. Вызов из модуля common.utils
2021-10-18 19:19:22,573 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1634573962.1940093, 'user': 'vlad'}
2021-10-18 19:19:22,576 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>, {'response': 202, 'data_list': []}) , {}. Вызов из модуля common.utils
2021-10-18 19:23:13,702 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1692, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 63578)>,) , {}. Вызов из модуля common.utils
2021-10-18 19:23:13,702 DEBUG core.py Getting data from client exception.
Traceback (most recent call last):
  File "H:\Программирование\Основной курс Python\Базы данных и PyQT\Project_Pyqt\QWW\server\core.py", line 87, in run
    get_message(client_with_message), client_with_message)
  File "H:\Программирование\Основной курс Python\Базы данных и PyQT\Project_Pyqt\QWW\common\decos.py", line 29, in log_saver
    ret = func_to_log(*args, **kwargs)
  File "H:\Программирование\Основной курс Python\Базы данных и PyQT\Project_Pyqt\QWW\common\utils.py", line 17, in get_message
    encoded_response = client.recv(MAX_PACKAGE_LENGTH)
ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
2021-10-18 19:23:13,703 INFO core.py Клиент ('127.0.0.1', 63578) отключился от сервера.
2021-10-18 20:03:27,721 DEBUG decos.py Была вызвана функция main c параметрами () , {}. Вызов из модуля __main__
2021-10-18 20:03:27,721 DEBUG decos.py Была вызвана функция config_load c параметрами () , {}. Вызов из модуля __main__
2021-10-18 20:03:27,722 DEBUG decos.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2021-10-18 20:03:27,722 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['H:/Программирование/Основной курс Python/Базы данных и PyQT/Project_Pyqt/Ready/server/server.py']
2021-10-18 20:03:27,723 DEBUG server.py Аргументы успешно загружены.
2021-10-18 20:03:28,469 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-10-18 20:05:05,949 INFO core.py Установлено соедение с ПК ('127.0.0.1', 56033)
2021-10-18 20:05:06,458 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>,) , {}. Вызов из модуля common.utils
2021-10-18 20:05:06,458 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1634576705.9570172, 'user': {'account_name': 'barsa', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzZ6g1mHMk5zRSLJ2u6qw\n5q5KXtc61/Q0OBiXkjiXnwT7jLCsXtYkxbnlDOdOJs/pMevIYhHCWdBfBhP4bW2K\nlt4Z3OZmbqoWma0avW/T/LZMLki0O0TIzCsLFhKK3hJw007Z3K0PDJHSshFsLBZr\nLjbOhltt0SK6/IfB+WJzegPvEH6iDOHb45GtMLUTQvigHYlzqzcQ5y5fP4dMihNv\ntqVpQ7PDwHOEKCw9FPSanL+LYjWQNdTWWN4VZCjIBxCWxUd0v2/tAIVYyw6dGYm5\nao3l8ylTk21uAQMdaNWikqbkNSsLs4s/NyHWQJmvfD3HdXEfZvwu1crnibSNaa5J\nJwIDAQAB\n-----END PUBLIC KEY-----'}}
2021-10-18 20:05:06,458 DEBUG core.py Start auth process for {'account_name': 'barsa', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzZ6g1mHMk5zRSLJ2u6qw\n5q5KXtc61/Q0OBiXkjiXnwT7jLCsXtYkxbnlDOdOJs/pMevIYhHCWdBfBhP4bW2K\nlt4Z3OZmbqoWma0avW/T/LZMLki0O0TIzCsLFhKK3hJw007Z3K0PDJHSshFsLBZr\nLjbOhltt0SK6/IfB+WJzegPvEH6iDOHb45GtMLUTQvigHYlzqzcQ5y5fP4dMihNv\ntqVpQ7PDwHOEKCw9FPSanL+LYjWQNdTWWN4VZCjIBxCWxUd0v2/tAIVYyw6dGYm5\nao3l8ylTk21uAQMdaNWikqbkNSsLs4s/NyHWQJmvfD3HdXEfZvwu1crnibSNaa5J\nJwIDAQAB\n-----END PUBLIC KEY-----'}
2021-10-18 20:05:06,459 DEBUG core.py Correct username, starting passwd check.
2021-10-18 20:05:06,461 DEBUG core.py Auth message = {'response': 511, 'bin': 'a350905a18b7f75df52a38efed50208192aeefcad5c43410a5080d09d9eafc75e72cbd91f02279a206bfec1b7c0f46ede4a903b8f254f3b11190d851461bf4fb'}
2021-10-18 20:05:06,461 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>, {'response': 511, 'bin': 'a350905a18b7f75df52a38efed50208192aeefcad5c43410a5080d09d9eafc75e72cbd91f02279a206bfec1b7c0f46ede4a903b8f254f3b11190d851461bf4fb'}) , {}. Вызов из модуля common.utils
2021-10-18 20:05:06,461 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>,) , {}. Вызов из модуля common.utils
2021-10-18 20:05:06,462 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>, {'response': 200}) , {}. Вызов из модуля common.utils
2021-10-18 20:05:07,096 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>,) , {}. Вызов из модуля common.utils
2021-10-18 20:05:07,096 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1634576706.4625788, 'account_name': 'barsa'}
2021-10-18 20:05:07,097 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>, {'response': 202, 'data_list': ['barsa']}) , {}. Вызов из модуля common.utils
2021-10-18 20:05:07,597 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>,) , {}. Вызов из модуля common.utils
2021-10-18 20:05:07,598 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1634576707.2297769, 'user': 'barsa'}
2021-10-18 20:05:07,602 DEBUG decos.py Была вызвана функция send_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>, {'response': 202, 'data_list': []}) , {}. Вызов из модуля common.utils
2021-10-18 20:14:24,311 DEBUG decos.py Была вызвана функция get_message c параметрами (<socket.socket fd=1680, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 56033)>,) , {}. Вызов из модуля common.utils
2021-10-18 20:14:24,311 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1634577263.8462975, 'account_name': 'barsa'}
2021-10-18 20:14:24,311 INFO core.py Клиент ('127.0.0.1', 56033) отключился от сервера.
